healthcareai/trained_models/trained_supervised_model.py

```diff
@@ -262,8 +266,12 @@ def make_factors(self, dataframe, number_top_features=3):
         # Run the raw dataframe through the preparation process
         prepared_dataframe = self.prepare_and_subset(dataframe)
 
-        # Create a new dataframe with the grain column from the original dataframe
-        results = dataframe[[self.grain_column]]
+        # Create a new dataframe. If grain column exists, add the grain column from the original dataframe; otherwise, 
+        # just create a new empty dataframe. 
```

Comment:
@danwellisch1 Appreciate this work! Can you please fix this comment so the lines stay within 80 characters?